lnd.go

```diff
@@ -187,7 +187,7 @@ type rpcListeners func() ([]*ListenerWithSignal, func(), error)
 // validated main configuration struct and an optional listener config struct.
 // This function starts all main system components then blocks until a signal
 // is received on the shutdownChan at which point everything is shut down again.
-func Main(cfg *Config, lisCfg ListenerCfg, shutdownChan <-chan struct{}) error {
+func Main(cfg *Config, lisCfg ListenerCfg, shutdownChan chan struct{}) error {
```

Comment:
`shutdownChan` can still be `<- chan struct` if you update `waitForWalletPassword` to also take a receive only channel?

Reply:
Thanks! Will fix